spec/eth/ens/resolver_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "spec_helper"
+
+describe Ens::Resolver do
+
+  let(:geth_dev_http_path) { "http://127.0.0.1:8545" }
```

Comment:
We shouldn't be making any http calls from tests. Look into `vcr` or `webmock` or simply putting a file with the expected response from the http endpoint in the `spec/` directory somewhere.

```suggestion
  let(:geth_dev_http_path) { "http://127.0.0.1:8545" }
```

Reply:
I've used both vcr and webmock, but given that we live in a time where we can easily run services in test environments, I tend to disagree. GHA can easily run a geth dev node, so... why not? If the container is updated to the latest version, the tests are now running against that (integration), which I think is a good thing. Also, I was just imitating behavior I saw in the existing test suite in an attempt to not stray too far off the beaten path.